src/index.ts

```diff
@@ -128,5 +128,3 @@ type LambdaInvokeError = {
     errorType: string;
     errorMessage: string;
 };
-
-export default app;
```

Comment:
That is a breaking change right?

Reply:
Ah, I didn't see usage of this default export anywhere, but I'm happy to keep this if you prefer.

Reply:
Fixed in 5c1add86781dfee6d685c751d3083c158826aa56

Reply:
Thanks! Since it's exported on NPM some people might use it via the programmatic API.